src/main/java/org/jenkinsci/plugins/workflow/flow/StashManager.java

```diff
@@ -109,26 +114,36 @@ public static void stash(@Nonnull Run<?,?> build, @Nonnull String name, @Nonnull
      * @see StashAwareArtifactManager#stash
      */
     @SuppressFBWarnings(value="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE", justification="fine if mkdirs returns false")
-    public static void stash(@Nonnull Run<?,?> build, @Nonnull String name, @Nonnull FilePath workspace, @Nonnull Launcher launcher, @Nonnull EnvVars env, @Nonnull TaskListener listener,
+    public static List<File> stash(@Nonnull Run<?,?> build, @Nonnull String name, @Nonnull FilePath workspace, @Nonnull Launcher launcher, @Nonnull EnvVars env, @Nonnull TaskListener listener,
```

Comment:
This is not binary compatible I think (changed return value).

Reply:
Olivier mention that in other PR, It will be changed and _overloaded_